packages/core/echo/automerge/src/automerge-repo.ts

```diff
@@ -2,4 +2,18 @@
 // Copyright 2023 DXOS.org
 //
 
+import { type BinaryDocumentId, type AutomergeUrl, stringifyAutomergeUrl } from '@automerge/automerge-repo';
+import * as Uuid from 'uuid';
+
 export * from '@automerge/automerge-repo';
+
+/**
+ * Returns a new Automerge URL with a random UUID documentId. Called by Repo.create(), and also used by tests.
+ * Copy-paste from automerge-repo.
+ * Copyright (c) 2019-2023, Ink & Switch LLC.
+ * Is not exported in @automerge/automerge-repo.
+ */
+export const generateAutomergeUrl = (): AutomergeUrl => {
```

Comment:
is this only exported for testing?

Reply:
createAutomergeUrl

Reply:
lets keep the same name as in automerge repo

Reply:
The name comes from the `automerge-repo` package